### ISSUE TITLE
Show attachment file size

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#271]
http://forums.zotero.org/discussion/15487/

Snapshots should show total of all files.

File size should be cached to avoid repeated filesystem hits.

**Comment by peadar-skoutelsky** [comment]
I agree with this issue, it's difficult to stay under the 300MB limit for free accounts without it. 
Especially when attaching annotated PDFs to items

**Comment by nixsee** [comment]
This would be very helpful. I'm somewhat surprised that it hasn't been implemented in the past 8 years...

**Comment by InfantLab** [comment]
Actually , we've been asking for this since 2009!! See https://forums.zotero.org/discussion/9169/sorting-items-by-size

To be honest I'm disappointed devs have done nothing about it. It must be easy to do. A cynical person might say that it's not a priority because it would revenues. (Allowing people to more easily stay below quotas). I don't see Zotero as that kind of project so curious why it hasn't been done.
